projects/plugins/protect/src/js/hooks/use-protect-data/index.js

```diff
@@ -22,10 +22,10 @@ export default function useProtectData() {
 	}
 
 	return {
-		numVulnerabilities: status.numVulnerabilities || 0,
-		numCoreVulnerabilities: status.core?.vulnerabilities?.length || 0,
-		numPluginsVulnerabilities: status.numPluginsVulnerabilities || 0,
-		numThemesVulnerabilities: status.numThemesVulnerabilities || 0,
+		numThreats: status.numThreats || 0,
+		numCoreThreats: status.core?.Threats?.length || 0,
```

Comment:
This was originally `status.core?.vulnerabilities?.length || 0`, should it now be `status.core?.threats?.length || 0` - no capitalization?

Reply:
Good catch @dkmyta! I've fixed this via [7cc5757](https://github.com/Automattic/jetpack/pull/25445/commits/7cc5757bab52eaebda9ecdd337455e8b89f71925) :+1: